packages/frontend/src/hooks/echarts/useEcharts.ts

```diff
@@ -753,6 +769,82 @@ const getEchartAxis = ({
     };
 };
 
+const getValidStack = (series: EChartSeries | undefined) => {
+    return series && (series.type === 'bar' || !!series.areaStyle)
+        ? series.stack
+        : undefined;
+};
+
+const calculateStackTotal = (
+    row: ResultRow,
+    series: EChartSeries[],
+    flipAxis: boolean | undefined,
+) => {
+    return series.reduce<number>((acc, s) => {
+        const hash = flipAxis ? s.encode?.x : s.encode?.y;
+        if (hash && row[hash]?.value.raw) {
+            acc += Number(row[hash]?.value.raw);
```

Comment:
how do you know value.raw is a number ? it could be a string or even a date... 
This might be causing the NaN 

![image](https://user-images.githubusercontent.com/1983672/201032091-909ac50c-6df4-4b25-aa27-ba47d73cb493.png)